### PR TITLE
Remove reference to itself

### PR DESCRIPTION
**Pull request by arnaldopereira**

### CODE REVIEW

**Comment by arnaldopereira-luizalabs** [comment]
I signed it!

File: docs/admin/namespaces/index.md [hunk 1]
```diff
@@ -37,8 +37,6 @@ The Namespace provides a unique scope for:
 
 ## Usage
```
Comment (johndmulhausen) [inline docs/admin/namespaces/index.md]: Need to remove this heading, too. And clear any excessive blank lines. :)

**Comment by arnaldopereira-luizalabs** [comment]
@johndmulhausen I've removed the heading, blank lines, fixed the size of the subsequent headings and squashed the commits. :)

**Comment by johndmulhausen** [comment]
Thanks, @arnaldopereira !

**Pull request merged**